<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>687</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>687</width>
    <height>414</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Slave Sucess V2.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Slave_Sucess/Fatcow-Farm-Fresh-Report-magnify.ico</normaloff>../Slave_Sucess/Fatcow-Farm-Fresh-Report-magnify.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">Qwidget::setFixedSize(587,334);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="prjButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prjLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="taskLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="slaveCkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Slave Task</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="microCkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Microstation Script</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="logCkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Export Log File</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Report Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="runSlaveCkBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Slave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exportsButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="logWindow">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(93, 93, 93);
margin: 0; 
padding: 0;
size: 8 px;</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;[TerraScan Project]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
   </widget>
   <widget class="QPushButton" name="reportsButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>119</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reportsLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>371</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>prjButton</zorder>
   <zorder>prjLineEdit</zorder>
   <zorder>taskLineEdit</zorder>
   <zorder>taskButton</zorder>
   <zorder>logCkBox</zorder>
   <zorder>microCkBox</zorder>
   <zorder>slaveCkBox</zorder>
   <zorder>logWindow</zorder>
   <zorder>reportsButton</zorder>
   <zorder>verifyButton</zorder>
   <zorder>reportsLineEdit</zorder>
   <zorder>progressBar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>prjButton</tabstop>
  <tabstop>taskButton</tabstop>
  <tabstop>slaveCkBox</tabstop>
  <tabstop>microCkBox</tabstop>
  <tabstop>logCkBox</tabstop>
  <tabstop>exportsButton</tabstop>
  <tabstop>prjLineEdit</tabstop>
  <tabstop>taskLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
